XCTWebDriverAgentLib/Commands/FBCustomCommands.m

```diff
@@ -51,4 +52,11 @@ + (NSArray *)routes
   return FBResponseDictionaryWithOK();
 }
 
++ (id<FBResponsePayload>)handleHideKeyboard:(FBRouteRequest *)request
+{
+    FBXCTSession *session = (FBXCTSession *)[FBSession activeSession];
+    [[session.application.windows elementBoundByIndex:0] tap];
```

Comment:
@marekcirkos Thanks for this. Right now it's actually just grabbing the Window and calling tap on it. My assumption was it's hitting it at (0,0) but I don't see in the docs for sure where it would hit, the docs just say "a hittable point". 

I'm sure this is not the ideal way, but based on what Apple is easily giving us, as a (hopefully) short-term solution to easily close the keyboard, we've just been doing it this way in our tests.

Are you ok with this?  I can definitely change it if desired. Let me know what you think.